packages/create-app/templates/default-app/app-config.yaml.hbs

```diff
@@ -69,11 +69,16 @@ catalog:
     # Backstage example templates
     - type: github
       target: https://github.com/spotify/backstage/blob/master/plugins/scaffolder-backend/sample-templates/react-ssr-template/template.yaml
+      allow: [Template]
```

Comment:
`allow` feels a bit confusing here. Can I use a noun here that isn't part of the catalog -> rule -> allow list?

Reply:
Addressing this in #2208